### DISCUSSION TITLE
Standalone flash player version 50.x.x installation on windows

### DISCUSSION THREAD

**Discussion by Gopal** [airsdk/Adobe-Runtime-Support#1417] [General]
I need to install the Harman flash player version 50.0.325 (standalone) on window machine. Please let me know from where I can download it and how to install?

**Comment by ajwfrost** [comment]
Hi
Flash Player version 50 is available via our enterprise licensing scheme - it's not a generally available binary that you can download from anywhere. If your company has a license agreement with us then you should contact whoever has been provided with the distribution: if you need to enter into a license agreement then please contact us via the "pbsupport at harman.com" mailbox.
thanks
